R/dt_options.R

```diff
@@ -19,72 +19,96 @@ dt_options_init <- function(data) {
-    "footnotes_border_top_color",         TRUE,  "footnote",         "value",   "#D3D3D3",
-    "footnote_sep",                      FALSE,  "footnote",         "value",   "<br />",
-    "footnote_marks" ,                   FALSE,  "footnote",         "values",  "numbers",
-    "footnote_font_size",                 TRUE,  "footnote",         "px",      "90%",
-    "sourcenote_padding",                 TRUE,  "sourcenote",       "px",      "4px",
-    "sourcenote_font_size",               TRUE,  "sourcenote",       "px",      "90%",
-    "sourcenotes_border_top_style",       TRUE,  "sourcenote",       "value",   "solid",
-    "sourcenotes_border_top_width",       TRUE,  "sourcenote",       "px",      "2px",
-    "sourcenotes_border_top_color",       TRUE,  "sourcenote",       "value",   "#D3D3D3",
+    "summary_row_border_style",           TRUE,  "summary_row",      "value",   "solid",
+    "summary_row_border_width",           TRUE,  "summary_row",      "px",      "2px",
+    "summary_row_border_color",           TRUE,  "summary_row",      "value",   "#D3D3D3",
+    "grand_summary_row_padding",          TRUE,  "grand_summary_row", "px",     "8px",
+    "grand_summary_row_background_color", TRUE,  "grand_summary_row", "value",  NA_character_,
+    "grand_summary_row_text_transform",   TRUE,  "grand_summary_row", "value",  "inherit",
+    "grand_summary_row_border_style",     TRUE,  "grand_summary_row", "value",  "double",
+    "grand_summary_row_border_width",     TRUE,  "grand_summary_row", "px",     "6px",
+    "grand_summary_row_border_color",     TRUE,  "grand_summary_row", "value",  "#D3D3D3",
+    "footnotes_font_size",                TRUE,  "footnotes",        "px",      "90%",
+    "footnotes_padding",                  TRUE,  "footnotes",        "px",      "4px",
+    "footnotes_background_color",         TRUE,  "footnotes",        "value",   NA_character_,
+    "footnotes_margin",                   TRUE,  "footnotes",        "px",      "0px",
+    "footnotes_border_style",             TRUE,  "footnotes",        "value",   "solid",
+    "footnotes_border_width",             TRUE,  "footnotes",        "px",      "2px",
+    "footnotes_border_color",             TRUE,  "footnotes",        "value",   "#D3D3D3",
+    "footnotes_sep",                     FALSE,  "footnotes",        "value",   "<br />",
+    "footnotes_marks" ,                  FALSE,  "footnotes",        "values",  "numbers",
+    "source_notes_padding",               TRUE,  "source_notes",     "px",      "4px",
+    "source_notes_background_color",      TRUE,  "source_notes",     "value",   NA_character_,
+    "source_notes_font_size",             TRUE,  "source_notes",     "px",      "90%",
+    "source_notes_border_style",          TRUE,  "source_notes",     "value",   "solid",
+    "source_notes_border_width",          TRUE,  "source_notes",     "px",      "2px",
+    "source_notes_border_color",          TRUE,  "source_notes",     "value",   "#D3D3D3",
```

Comment:
[No action needed] I suspect that in the future, when we get more serious about RTF and LaTeX, we'll need to stop populating the table so completely at startup, so we can tell the difference between what options the user has explicitly chosen and which should have the default values; this will let us warn if the user has explicitly chosen options that cannot be supported in RTF/LaTeX.